internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriterOptions.java

```diff
@@ -127,6 +143,9 @@ private void addDefaults() {
       if (time == null) {
         time = SystemTime.INSTANCE;
       }
+      if (maxRecordSizeBytes == null) {
+        maxRecordSizeBytes = -1;
```

Comment:
Nit,
Shall we replace `-1` by `VeniceWriter.UNLIMITED_MAX_RECORD_SIZE`?

Reply:
Nice catch. Fixed.